cpp/src/slice2swift/Gen.cpp

```diff
@@ -1302,9 +1290,7 @@ Gen::ValueVisitor::visitClassDefStart(const ClassDefPtr& p)
     // else inherit Value's initializer.
 
     out << sp;
-    out << nl << "/// Returns the Slice type ID of the interface supported by this object.";
-    out << nl << "///";
-    out << nl << "/// - returns: `String` - The Slice type ID of the interface supported by this object.";
+    out << nl << "/// - Returns: The Slice type ID of the interface supported by this object.";
```

Comment:
When the overview and returns section say the exact same thing, it's fine to just have a lone `Returns` section.